lldb/test/API/terminal/TestEditline.py

```diff
@@ -44,3 +44,15 @@ def test_left_right_arrow(self):
             )
 
         self.quit()
+
+    @skipIfAsan
+    @skipIfEditlineSupportMissing
+    def test_prompt_color(self):
+        """Test that ANSI escape codes don't break the prompt"""
+        self.launch(use_colors=True)
```

Comment:
Can we also test with "use_colors=False" to ensure these sequences don't get inserted?